[32mINFO    [0m root:task_utility.py:14 Executing create task api
[32mINFO    [0m root:task_utility.py:15 Sending the POST request http://localhost:5000/tasks
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:226 Starting new HTTP connection (1): localhost:5000
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 http://localhost:5000 "POST /tasks HTTP/1.1" 200 23
[32mINFO    [0m root:task_utility.py:18 Got the status  code as 200
[32mINFO    [0m root:task_utility.py:24 {'task_id': 'LowXWjbN'}
[32mINFO    [0m root:task_utility.py:25 Created Task_id is :: LowXWjbN
[32mINFO    [0m root:test_task_update.py:19 Executing Modify Task API
[32mINFO    [0m root:test_task_update.py:20 Sending the POST request with task id LowXWjbN
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:226 Starting new HTTP connection (1): localhost:5000
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 http://localhost:5000 "PUT /tasks/LowXWjbN HTTP/1.1" 200 23
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:226 Starting new HTTP connection (1): localhost:5000
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 http://localhost:5000 "GET /tasks/LowXWjbN HTTP/1.1" 200 44
[32mINFO    [0m root:test_task_update.py:22 Verify update task json is updated successfully expected {'task': 'modifiedtask', 'completed': False} ,Actual {'task': 'modifiedtask', 'completed': False}